packages/help-center/src/components/help-center-feedback-form.tsx

```diff
@@ -69,7 +70,11 @@ const HelpCenterFeedbackForm = ( { postId, blogId, slug }: HelpCenterFeedbackFor
 
 	return (
 		<div className="help-center-feedback__form">
-			{ startedFeedback === null ? <FeedbackButtons /> : <FeedbackTextArea /> }
+			{ startedFeedback === null && <FeedbackButtons /> }
+			{ startedFeedback !== null && answerValue === 1 && <FeedbackTextArea /> }
+			{ startedFeedback !== null && answerValue === 2 && (
+				<HelpCenterContactSupportOption trackEventName="calypso_helpcenter_feedback_contact_support" />
```

Comment:
Since there is no context for the HE where this support session is created maybe we should pass an initial message to Zendesk with an explanation. Something brief with a link to the article the user was viewing. Otherwise I could see the user trying to reference the article in the support message and the HE having no way to understand.

Reply:
That's a great idea @arcangelini